### DISCUSSION TITLE
[Feature] Android-App register as share app to upload

### DISCUSSION THREAD

**Discussion by Dschogo** [immich-app/immich#4638] [Feature Request]
### The feature

Hey there,

Would be nice to register immich as an app to "share images/videos". 

Why? 
Imagine we have a photo we want to backup which isn't in the included folders (like whatsapp images - don't want to backup all)
Sometimes there is one I want to backup - then I'd love to just open the native gallery app, select the photos(s) and then share to upload. Similar to Goolge Photos

Happy to hear if there is already a way to do this without the mobile webbrowser

As always with FOSS love to y'all ❤️ 

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #2659
